src/main/java/org/breedinginsight/brapps/importer/services/MappingManager.java

```diff
@@ -279,7 +291,9 @@ private void mapField(Object parent, Field field, List<MappingField> mappings, T
 
                 // Check non-null value
                 if (required != null && fileValue.isBlank()) {
-                    throw new UnprocessableEntityException(String.format(blankRequiredField,  matchedMapping.getValue().getFileFieldName()));
+                    //throw new UnprocessableEntityException(String.format(blankRequiredField,  matchedMapping.getValue().getFileFieldName()));
```

Comment:
Did you want this commented out code to remain?

Reply:
Thanks, had that in for reference when working and forgot to remove